src/components/team/styledTeam.js

```diff
@@ -0,0 +1,61 @@
+import styled from "styled-components";
+
+export const TeamWrapper = styled.section`
+    display: flex;
+    flex-direction: column;
+    width: 53.61%;
+    margin: auto;
+    padding: 150px 0;
```

Comment:
I'm seeing the padding bottom as 136px. As far as padding top goes, I think i included it in the prices section. Might need to be something like padding 0 0 136px;